modules/core/src/lifecycle/props.js

```diff
@@ -170,8 +170,10 @@ function diffUpdateTriggers(props, oldProps) {
 }
 
 function diffUpdateTrigger(props, oldProps, triggerName) {
-  const newTriggers = props.updateTriggers[triggerName] || {};
-  const oldTriggers = oldProps.updateTriggers[triggerName] || {};
+  let newTriggers = props.updateTriggers[triggerName];
```

Comment:
why do we have a legitimate "newTriggers === 0"  case?

Reply:
@jianhuang01 
In the demo example, the updateTriggers is defined as `{getColor: this.state.colorIndex}`. When calling [diffUpdateTrigger](https://github.com/uber/deck.gl/blob/master/modules/core/src/lifecycle/props.js#L173-L174)
`updateTriggers[triggerName]` may return 0.